client/src/i18n/fr/form.json

```diff
@@ -79,7 +79,7 @@
             "CONFIRM_ACTION": "Confirmez vous cette action ?",
             "CONFIRMATION_NEEDED": "Confirmation Requise",
             "CONFIRM_ACTIVATION": "Voulez vous vraiment activer les droits d'accès de cet utilisateur",
-            "CONFIRM_CREDIT_NOTE": "La facture dont vous voulez annuler a déjà été payée, voulez-vous vraiment annuler cette facture ",
+            "CONFIRM_CREDIT_NOTE": "Attention! La facture dont vous voulez annuler a déjà été payée! Voulez-vous vraiment annuler cette facture?",
```

Comment:
The correct corresponding translation in French is : 
`Attention! La facture que vous avez sélectionnée a déjà été payée! L'inverser peut entraîner des déséquilibres dans votre comptabilité. Êtes-vous sûr de vouloir continuer à annuler cette facture?`